compiler/Cargo.toml

```diff
@@ -12,15 +12,15 @@ description = "Compiler lib for the Fe language."
 solc-backend = ["solc"]
 
 [dependencies]
+fe-common = {path = "../common", version = "0.0.1"}
 fe-parser = {path = "../parser", version = "0.0.1"}
 fe-semantics = {path = "../semantics", version = "0.0.1"}
 serde_json = "1.0"
 serde = "1.0"
 hex = "0.4"
 # This fork contains the shorthand macros and some other necessary updates.
-yultsur = { git = "https://github.com/g-r-a-n-t/yultsur" }
+yultsur = { git = "https://github.com/cburgdorf/yultsur", branch="christoph/feat/data-support"}
```

Comment:
This depends on https://github.com/g-r-a-n-t/yultsur/pull/1 and needs to be updated when its merged.

Reply:
merged